docs/UserGuide.adoc

```diff
@@ -552,12 +568,38 @@ image::showgroupcommand.png[]
 image::showgroupresult.png[]
 
 *****
-image:./icons/lightbulb.png[] You can use the command `n` to help you scroll the schedule view without touching your mouse!
+image:./icons/lightbulb.png[] You can use the command `scroll` to help you scroll the schedule view without touching your mouse!
 
-image:./icons/lightbulb.png[] You can use the command `nw` to help you show next week's schedule!
+image:./icons/lightbulb.png[] You can use the command `togglenext` to help you show next week's schedule!
+After you've reached the fourth week's schedule, the next `togglenext` command will reset the schedule view back to the first week's schedule.
 
 *****
 
+==== Look at some members: `lookat`
+Suppose that all of your group members are extremely busy and the common free time is too short to do any productive work,
+you may use the `lookat` command to see if you can meet with some of your group members. You must be viewing a group's
+schedule in order for this command to work.
+
+Format:
+
+* `lookat [n/Name]...`
+
+The `lookat` command takes in any number of `n/Name` and highlights the group members according to name.
+
+Example:
+
+Suppose your group schedule looks like the one below.
+//Image1
+
+* Simply type `lookat n/Bobby n/Denise" if you are interested in inspecting both of their schedules!
```

Comment:
I think there's a formatting error here regarding the backtick and inverted commas in `` `lookat n/Bobby n/Denise"``.

Reply:
Thanks for pointing out! fixed!